gui/qt/util.py

```diff
@@ -63,6 +63,8 @@ def __init__(self, text, func):
     def keyPressEvent(self, e):
         if e.key() == Qt.Key_Return:
             self.func()
+        else:
+            e.ignore()
```

Comment:
Huh? Why? It's typical for keyboard navigation to work... why do this?

Reply:
This actually should be calling `QPushButton.keyPressEvent(e)` if it intends to ignore here.  Can you elaborate on why this change is necessary and/or what it wants to accomplish?  EnterButton is used everywhere...

But yeah the Electrum programmers that wrote this didn't read the Qt docs.  Read this:  https://doc.qt.io/qt-5/qwidget.html#keyPressEvent

You are supposed to call the super class impl. if you intend to ignore it...  And if you don't ignore it call the base class... What's happening here is that the e.ignore() is pretty much superfluous (I think).  Let me know what this fixes... Thanks.

Reply:
Oh.. I just read your commit message.  I think the functionality you want will be accomplished by calling `QPusButton.keyPressEvent(e)` (as per the Qt docs)... lmk.